src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+
+  if (days >= 7) {
+    return days * dayPrice - 50;
+  }
```

Comment:
It is better to avoid using 'if else' structures when each 'if' block ends with a 'return' statement. This can simplify the code structure.